src/Pages/Detail/Components/Community.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import axios from "axios";
+import Button from "./Button";
+import CommunityPost from "./CommunityPost";
+import CreateMessage from "./CreatorMessage";
+import { API } from "./Utils";
+const Community = ({ focusTarget }) => {
+  const [more, setMore] = useState(false);
+  const [community, setCommunity] = useState([]);
+
+  const getComment = () => {
+    axios.get(API).then((res) => setCommunity(res.data.community));
+    setMore(true);
+  };
+
+  useEffect(() => {
+    axios.get(API).then((res) => setCommunity(res.data.community.slice(0, 5)));
+  }, []);
+
+  return (
+    <CommunityWrap>
+      <div className="header" ref={(el) => (focusTarget.current[1] = el)}>
+        <div className="title">
+          <h2>커뮤니티</h2>
+          <p>120개의 글</p>
+        </div>
+        <Button bgcolor="#3e4042" width="130px" weight="200">
+          글 작성하기
+        </Button>
+      </div>
+      <CreateMessage />
+      {community?.map(
+        ({ profile_image, nickname, date, description, comments }, i) => (
+          <CommunityPost
+            key={i}
+            img={profile_image}
+            comments={comments}
+            nickname={nickname}
+            date={date}
+          >
+            {description}
+          </CommunityPost>
+        ),
+      )}
```

Comment:
```suggestion
        (post, idx) => (
          <CommunityPost
            key={idx}
            img={post.profile_image}
            comments={post.comments}
            nickname={post.nickname}
            date={post.date}
          >
            {description}
          </CommunityPost>
        ),
      )}
```
이 경우에는 구조분해 할당 안하는 것이 더 나아보이기도 하네요